### ISSUE TITLE
can not create new run: get() returned more than one User -- it returned 2!

### ISSUE THREAD

**Issue by jhutar** [Nitrate/Nitrate#327]
```
Environment:


Request Method: POST
Request URL: http://localhost:8000/run/new/

Django Version: 1.11.13
Python Version: 2.7.15
Installed Applications:
('django.contrib.admin',
 'django.contrib.admindocs',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.messages',
 'django.contrib.sessions',
 'django.contrib.sites',
 'django.contrib.staticfiles',
 'django_comments',
 'kobo.django.xmlrpc',
 'tinymce',
 'tcms.core.contrib.auth.apps.AppConfig',
 'tcms.core.contrib.comments.apps.AppConfig',
 'tcms.core.contrib.linkreference',
 'tcms.core.logs',
 'tcms.integration.bugzilla',
 'tcms.integration.errata',
 'tcms.management',
 'tcms.profiles',
 'tcms.testcases',
 'tcms.testplans',
 'tcms.testruns',
 'tcms.xmlrpc.apps.AppConfig',
 'tcms.core.apps.AppConfig',
 'debug_toolbar',
 'django_extensions')
Installed Middleware:
('django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.locale.LocaleMiddleware',
 'django.middleware.common.CommonMiddleware',
 'tcms.core.middleware.CsrfDisableMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'debug_toolbar.middleware.DebugToolbarMiddleware')



Traceback:

File "/home/pok/Checkouts/Nitrate/.venv/lib/python2.7/site-packages/django/core/handlers/exception.py" in inner
  41.             response = get_response(request)

File "/home/pok/Checkouts/Nitrate/.venv/lib/python2.7/site-packages/django/core/handlers/base.py" in _legacy_get_response
  249.             response = self._get_response(request)

File "/home/pok/Checkouts/Nitrate/.venv/lib/python2.7/site-packages/django/core/handlers/base.py" in _get_response
  187.                 response = self.process_exception_by_middleware(e, request)

File "/home/pok/Checkouts/Nitrate/.venv/lib/python2.7/site-packages/django/core/handlers/base.py" in _get_response
  185.                 response = wrapped_callback(request, *callback_args, **callback_kwargs)

File "/home/pok/Checkouts/Nitrate/.venv/lib/python2.7/site-packages/django/views/decorators/http.py" in inner
  40.             return func(request, *args, **kwargs)

File "/home/pok/Checkouts/Nitrate/.venv/lib/python2.7/site-packages/django/contrib/auth/decorators.py" in _wrapped_view
  23.                 return view_func(request, *args, **kwargs)

File "/home/pok/Checkouts/Nitrate/tcms/testruns/views.py" in new
  117.         if form.is_valid():

File "/home/pok/Checkouts/Nitrate/.venv/lib/python2.7/site-packages/django/forms/forms.py" in is_valid
  183.         return self.is_bound and not self.errors

File "/home/pok/Checkouts/Nitrate/.venv/lib/python2.7/site-packages/django/forms/forms.py" in errors
  175.             self.full_clean()

File "/home/pok/Checkouts/Nitrate/.venv/lib/python2.7/site-packages/django/forms/forms.py" in full_clean
  384.         self._clean_fields()

File "/home/pok/Checkouts/Nitrate/.venv/lib/python2.7/site-packages/django/forms/forms.py" in _clean_fields
  402.                     value = field.clean(value)

File "/home/pok/Checkouts/Nitrate/tcms/core/forms/fields.py" in clean
  47.                         (Q(email=value) | Q(username=value)))

File "/home/pok/Checkouts/Nitrate/.venv/lib/python2.7/site-packages/django/db/models/manager.py" in manager_method
  85.                 return getattr(self.get_queryset(), name)(*args, **kwargs)

File "/home/pok/Checkouts/Nitrate/.venv/lib/python2.7/site-packages/django/db/models/query.py" in get
  384.             (self.model._meta.object_name, num)

Exception Type: MultipleObjectsReturned at /run/new/
Exception Value: get() returned more than one User -- it returned 2!
```

**Comment by jhutar** [comment]
Oh, this is probably I have two users with same email

**Comment by tkdchen** [comment]
@jhutar How did the email go into Nitrate? Input from admin page or just from `User` API?

**Comment by jhutar** [comment]
Sorry, I do not remember exactly. I probably just created two different users for testing of reporting but because I wanted to see emails for both of them, I have set their emails to mine email.

**Comment by tkdchen** [comment]
Fixed in issue #430
